assets/src/edit-story/components/button/index.js

```diff
@@ -126,3 +126,13 @@ export const GridView = ( props ) => (
 		<GridViewIcon />
 	</StyledButton>
 );
+
+export const ActionButton = styled.button.attrs( { type: 'button' } )`
+	background: transparent;
+	border: 1px solid ${ ( { theme } ) => theme.colors.fg.v3 };
+	border-radius: 4px;
+	color: ${ ( { theme } ) => theme.colors.bg.v5 };
+	font-size: 15px;
+	line-height: 30px;
+	padding: 0 15px;
```

Comment:
Replying to https://github.com/ampproject/amp-wp/pull/4138#discussion_r372117904:

I merely moved the button - it still existed with the same design before. And actual design wasn't implemented. But I have now implemented the "real" design or as close as we can get right now.